README.md

```diff
@@ -0,0 +1,21 @@
+# ssl-refbox
+RoboCup Small Size League Referee Box
+
+
+to install:
+
+1: clone the repro somewhere
+
+2: enter the directory of the git repro and open a terminal there.
+
+3: in a terminal type: ./installDeps.sh
+
+4: create a build driectory: To do so type "mkdir build" in the terminal
```

Comment:
This and the following steps are already contained in the `Makefile`. Simply running `make` will automatically create the build directory, run cmake and make in the build folder.

Reply:
Simply run `make` to build the project. No need to deal with cmake, that's done by the Makefile. Try it out ;)